src/enterprise/requests/UserInfo.ts

```diff
@@ -1,11 +1,15 @@
 import { tabNineProcess } from "../../binary/requests/requests";
 
+type KnownScopes = "chat" | "completions";
+
 export type UserInfo = {
   email: string;
   team: [];
   verified: [];
   isLoggedIn: boolean;
+  scopes?: KnownScopes[];
```

Comment:
It's not optional in binary if I remember correctly

Reply:
It is for backward compitability